docs/web-code-style.md

```diff
@@ -0,0 +1,20 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+   -
+   - SPDX-License-Identifier: CC0-1.0
+   -->
+
+As foundation for our TypeScript/SCSS styleguide
+[this styleguide](!https://basarat.gitbook.io/typescript/styleguide) is used,
+except for the followings:
```

Comment:
Why do we make this exception btw?

Reply:
Because we usually have every React component in a separate file (like Java class) and we usually name them in a way: `DeletionDialog`, `ModalPopup`, or something like this. We don't want to have it `deleteionDialog` and `modalPopup`